scripts/onboarding/aks/onboarding-using-msi-auth/existingClusterParam.json

```diff
@@ -59,6 +59,15 @@
           "<existingOrnew-tag-name2>": "<existingOrnew-tag-value2>",
           "<existingOrnew-tag-nameN>": "<existingOrnew-tag-valueN>"
         }
+      },
+      "dataCollectionInterval":  {
+        "value" : "1m"
+      },
+      "nameSpacesFilteringModeForDataCollection": {
+        "value": "Off"
+      },
+      "namespacesForDataCollection": {
+        "value": [ "kube-system", "gatekeeper-system" ]
```

Comment:
I believe we agreed to include "azure-arc" namespace to default namespace list for aks and arc? that's what I'm doing with the cost presets in the UX so just flagging this to be consistent

Reply:
> I believe we agreed to include "azure-arc" namespace to default namespace list for aks and arc? that's what I'm doing with the cost presets in the UX so just flagging this to be consistent

> In templates, by default namespace filtering mode is "Off" and namespaces doesn't applied to the filtering unless customer changes filtering mode to include or exclude.  lets sync offline and see what are the settings you are using for profiles and probable we can have templates to have the cost profile as parameter which will control the settings.

Reply:
included "azure-arc" to the default namespace list to have consistent with ux.